issuer.yml

```diff
@@ -72,6 +72,43 @@ components:
   securitySchemes:
     $ref: "./components/SecuritySchemes.yml#/components/securitySchemes"
   schemas:
+    UpdateCredentialStatusRequest:
+      type: object
+      required: ['credentialId', 'credentialStatus', 'status']
+      additionalProperties: false
+      description: Credential status information to be updated.
+      properties:
+        credentialId:
+          type: string
+          description: Identifies the credential (the identifier does not have to appear in the VC itself).
+        credentialStatus:
+          type: object
+          description: Identifies the specific status list entry to update.
+          required: ['type', 'statusPurpose']
+          additionalProperties: false
+          properties:
+            id: 
+              type: string
+            type: 
+              type: string
+            statusPurpose:
+              type: string
+            statusListIndex:
+              type: string
+            statusListCredential:
+              type: string
+        status:
+          type: boolean
+          description: Specifies the new status.
+        indexAllocator:
```

Comment:
This feature needs to be explained more clearly in the specification (in a future PR).